packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFailedPaymentsCancellation/index.txt

```diff
@@ -0,0 +1,7 @@
+subscriptionFailedPaymentsCancellation-subject = "Your <%- productName %> subscription has been cancelled"
+
+subscriptionFailedPaymentsCancellation-title = "Your subscription has been cancelled"
+
+subscriptionFailedPaymentsCancellation-content = " We've cancelled your <%- productName %> subscription because multiple payment attempts failed. To get access again, start a new subscription with an updated payment method."
```

Comment:
The apostrophe in this `We've` needs to be updated too.

Nitpick: Can you remove the space before `We've` as well?